### ISSUE TITLE
Running multiple nvidia containers at a time

### ISSUE THREAD

**Issue by jithum77** [NVIDIA/nvidia-docker#729]
Can I run multiple nvidia containers in parallel? 

eg: docker run --runtime=nvidia --rm gpu_based_container container_cmd
docker run --runtime=nvidia --rm gpu_based_container container_cmd

**Comment by jithum77** [comment]
Hello Renaud, thanks for the quick response, who manages the concurrent gpu device access, should this be handled properly within the container or is it done on the nvidia driver layer?

**Comment by jithum77** [comment]
I have two containers running in parallel, trying to access the same GPU
device in the host system, this is my scenario, when I executed, the first
container got mapped to the CuDa device successfully, but the second throws
an error which is related to device access problem. Do you see any problem
in this scenario?

Anyhow, finally I need to run multiple containers with each of them
accessing GPU device on the host. Thanks

**Comment by dimorinny** [comment]
I'm trying to run gpu accelerated android emulators with tesla p40.

But I've faced with OpenGL rendering errors inside emulators. Mostly, that errors about allocating frame buffers.

I found, that frequency of frame buffer allocating errors depends on emulators count, that use shared GPU card. (for example, everything work great with 3 emulators, but I have a lot of errors, when I have, 6 emulators).

The question: Is it secure to share GPU between containers?
